web/src/pages/project/SeqrSync.tsx

```diff
@@ -101,38 +116,59 @@ const SeqrSync: React.FunctionComponent<SeqrSyncProps> = ({ syncTypes, project }
                         <Form.Checkbox
                             id="syncIndividuals"
                             checked={syncOptions.syncIndividuals}
-                            onChange={updateStateFromForm}
+                            onChange={updateStateFromCheckbox}
                             label="Sync pedigree"
                         />
                         <Form.Checkbox
                             id="syncFamilies"
                             checked={syncOptions.syncFamilies}
-                            onChange={updateStateFromForm}
+                            onChange={updateStateFromCheckbox}
                             label="Sync families"
                         />
                         <Form.Checkbox
                             id="syncIndividualMetadata"
                             checked={syncOptions.syncIndividualMetadata}
-                            onChange={updateStateFromForm}
+                            onChange={updateStateFromCheckbox}
                             label="Sync individuals metadata"
                         />
                         <Form.Checkbox
```

Comment:
Just considering some UX here, could you indent the esIndexTypes to show they're grouped. Just throwing out these ideas:

- Disabling the syncESIndex checkbox disables the esIndexTypes and shows an empty list
- (less keen on) removing the checkbox and only having the multi-select. Even though it's simpler, I think it actually makes it more annoying to unselect a specific index.